### PR TITLE
Big pr

### PR DESCRIPTION
**Pull request by nirzalcman**

### CODE REVIEW

File: src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackCreator.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.hibob.academy.employeeFeedback.Service
+
+import com.hibob.academy.employeeFeedback.dao.CreationFeedback
+import com.hibob.academy.employeeFeedback.dao.FeedbackDao
+import com.hibob.academy.employeeFeedback.dao.UserLoggedInDetails
+
+
+class FeedbackCreator(private val feedbackDao: FeedbackDao) {
+    fun createFeedBack(userLoggedInDetails: UserLoggedInDetails, feedback: CreationFeedback): Long {
```
Comment (rakreshet) [inline src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackCreator.kt]: `FeedbackCreationRequest` is a better name, but you can rename it later in a different PR. I don't want it to stop you

File: src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackCreator.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.hibob.academy.employeeFeedback.Service
+
+import com.hibob.academy.employeeFeedback.dao.CreationFeedback
+import com.hibob.academy.employeeFeedback.dao.FeedbackDao
+import com.hibob.academy.employeeFeedback.dao.UserLoggedInDetails
+
+
+class FeedbackCreator(private val feedbackDao: FeedbackDao) {
```
Comment (rakreshet) [inline src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackCreator.kt]: I'm not sure how it is in kotlin, but class should have an annotation like @Singleton we use in scala?

File: src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackUpdateor.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package com.hibob.academy.employeeFeedback.Service
+
+
+import com.hibob.academy.employeeFeedback.dao.*
+import com.hibob.academy.employeeFeedback.dao.FeedbackDao
+import jakarta.ws.rs.NotFoundException
+
+class FeedbackUpdateor(private val feedbackDao: FeedbackDao) {
```
Comment (rakreshet) [inline src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackUpdateor.kt]: I think there's a spelling mistake here, right?

File: src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackUpdateor.kt [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+    fun updateStatus(companyId: Long, status: Status, feedbackId: Long): Boolean =
```
Comment (rakreshet) [inline src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackUpdateor.kt]: Who is allow to update feedback status? where is the role validation?

File: src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackCreator.kt (hunk 1 above)
Comment (nirzalcman) [reply src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackCreator.kt]: i will do it in another pr later .

File: src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackUpdateor.kt (hunk 4 above)
Comment (nirzalcman) [reply src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackUpdateor.kt]: did it in annotation.

**Pull request closed without merging**